mm2src/coins/lp_coins.rs

```diff
@@ -4998,6 +5011,32 @@ pub async fn delegations_info(ctx: MmArc, req: DelegationsInfo) -> Result<Json,
 
             qtum.get_delegation_infos().compat().await.map(|v| json!(v))
         },
+
+        DelegationsInfoDetails::Cosmos(r) => match coin {
+            MmCoinEnum::Tendermint(t) => Ok(t.delegations_list(r.paging).await.map(|v| json!(v))?),
+            MmCoinEnum::TendermintToken(t) => Ok(t.platform_coin.delegations_list(r.paging).await.map(|v| json!(v))?),
```

Comment:
I think we should return an error for tendermint tokens as they can't be delegated